Description: Set SYMPOW_CACHEDIR to be inside DOT_SAGE
 Otherwise tests fail in sbuild because sympow can't create HOME/.sympow
Author: Ximin Luo <infinity0@debian.org>
Forwarded: TODO
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage/lfunctions/sympow.py
+++ b/sage/src/sage/lfunctions/sympow.py
@@ -50,6 +50,7 @@
 
 import os
 
+from sage.env import DOT_SAGE
 from sage.structure.sage_object import SageObject
 from sage.misc.all import pager, verbose
 import sage.rings.all
@@ -77,7 +78,7 @@
         """
         Used to call sympow with given args
         """
-        cmd = 'env -u MALLOC_PERTURB_ sympow %s' % (DOT_SAGE, args)
+        cmd = 'env -u MALLOC_PERTURB_ SYMPOW_CACHEDIR="%s/sympow///" sympow %s' % (DOT_SAGE, args)
         with os.popen(cmd) as f:
             v = f.read().strip()
         verbose(v, level=2)
